apis/trello/tool.gpt

```diff
@@ -0,0 +1,13 @@
+Name: Trello API
+Share Credential: trello-api-key as api.trello.comAPIKey
+Share Credential: github.com/gptscript-ai/credential as api.trello.comAPIToken with GPTSCRIPT_API_TRELLO_COM_APITOKEN as env and "Please go to [here](https://trello.com/1/authorize?expiration=never&scope=read,write,account&response_type=token&key=ef4fde9fd7f65a15c74322f7546e4dee) to authenticate and provide your trello API token" as message and key as field
+Share Tools: *boards* from ./openapi.json
+Share Tools: *cards* from ./openapi.json
+Share Tools: *organizations* from ./openapi.json
+
+---
+Name: trello-api-key
+
+#!/usr/bin/env bash
+
+echo "{\"env\":{\"GPTSCRIPT_API_TRELLO_COM_APIKEY\":\"ef4fde9fd7f65a15c74322f7546e4dee\"}}"
```

Comment:
Is this key meant to be in here?

Reply:
This is public API key and it is safe be to here. It identifies the auth app. 
>Tokens for users should always be securely stored as they grant access to the entire user's account! It is ok for your API key to be publicly available, but a token should never be publicly available. If a token becomes public, it should be revoked immediately by the user.